scrape.py

```diff
@@ -27,6 +28,14 @@ def get_duration(seconds):
     minutes, seconds = divmod(seconds, 60)
     return f"{minutes} mins {seconds} secs"
 
+def fix_potential_links(entry):
```

Comment:
This feels like a lot of extra faff and maintenance, for something [`html2text`](https://pypi.org/project/html2text/) can just do magically, and handle all the edge cases. Someone else's code always wins. I've used the package before, it's really nice for this kind of thing.

Reply:
Done :)